src/ini.cr

```diff
@@ -21,7 +21,7 @@ class INI
   #
   # INI.parse("[foo]\na = 1") # => {"foo" => {"a" => "1"}}
   # ```
-  def self.parse(str) : Hash(String, Hash(String, String))
+  def self.parse(str : String) : Hash(String, Hash(String, String))
```

Comment:
I think it could be an IO too

Reply:
I was wondering too, if the same method can be used.
I've added a test.